portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,21 @@ function addRandomFunFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+function getRandomGreeting() {
+    const greetings =
+      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+
+    // Pick a random greeting.
+    var greeting = greetings[Math.floor(Math.random() * greetings.length)];
+    return greeting;
+}
+
+/**
+ * Fetches data from server and adds it to home page.
+ */
+function onloader(){
+  fetch("/data").then(response => response.json()).then((infoAboutMe) => {
+    document.getElementById("Introduction").append(`${getRandomGreeting()} ${infoAboutMe[0]} ${infoAboutMe[1]} ${infoAboutMe[2]}`)
```

Comment:
what happens if you add an extra piece of info to infoAboutMe, or remove one from above?
I think instead of hardcoding the index numbers here, it would be better to call out to another function that returns your infoAboutMe array in whatever string format you want

Reply:
Yes makes sense I have added a function for that.